### ISSUE TITLE
Merge editor accessibility improvements

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#159029]
Let's save some time in September to think more about merge editor accessibility.
Some ideas:

1. Accept Merge button should be a tab stop @jrieken So the user should go to the result view, press tab and have focus on the button
2. We should play audio cues when the user navigates to a line in 3-way merge editor. We could play different sounds based on:
* Line is resolved
* Line is not resolved
3. All the checkbox / context menu actions need to be available in the command palette so users can easily access them
4. 3-way merge aria label of the editor could end with some helpful tips

Ideas welcome :) 

fyi @daviddossett

**Comment by daviddossett** [comment]
Once we have the ability to navigate through conflicts [like this](https://github.com/microsoft/vscode/issues/159027#issuecomment-1226485407), how does a screen reader announce each conflict/side? I'm sure we have similar challenges with simple diff views, but I'm curious how we explain word diffs and things like that.

**Comment by isidorn** [comment]
@daviddossett that's a good question. If the navigation puts the focus on the left side the screen reader would just read the focused line - without a lot of context.
I have to think more about this. Since the better screen reader experience might be the old "everything in one editor".

**Comment by isidorn** [comment]
I do not think this will happen this milestone. Thus moving to November.
Un-assigning myself. @hediet feel free to reach out if I can help somehow here.

fyi @meganrogge

**Comment by jooyoungseo** [comment]
@meganrogge -- Very recently, I have encountered accessibility issue with merge editor when trying to sync my VSCode setting conflicts between local and remote ones. It would be so nice if this editor could provide similar experience to the diff editor audio cues, etc.

It would be even nicer if some frequently used buttons, such as Accept, had their default accelerator shortcut key so that users can quickly activate the buttons without losing their current focus in the editor.

**Comment by isidorn** [comment]
@meganrogge it makes sense that the default actions have always the same keybindings. Though there have not been discussion like this in the past.
If this is coming from notifications I suggest to reach out to Ben and get his thoughts. I think from the notification service perspective, every client contributes different buttons.

**Comment by meganrogge** [comment]
@bpasero do you have thoughts here about having some notification contribution pt that we call `accept` for example and then have an action with a keybinding that will choose this option?

**Comment by bpasero** [comment]
Native dialogs typically solve this with support for mnemonics: 

<img width="168" alt="image" src="https://user-images.githubusercontent.com/900690/220439288-e32da301-3a78-4ce6-af26-451962c23ece.png">

We assign those for Windows and Linux, I think macOS does not really have such a concept.

This makes sense in a modal context where you need to act on the dialog anyways, not sure about notifications though. We do consider the very first button in a notification as the primary button and thus show it in blue:

<img width="165" alt="image" src="https://user-images.githubusercontent.com/900690/220439495-36b97a57-53ac-49be-b5d2-4d0408767973.png">

We could add a command that simply triggers the primary button. But how would it work if multiple notification toasts are visible?

**Comment by meganrogge** [comment]
Maybe the command would only trigger the primary button when there's only one notification visible?

**Comment by bpasero** [comment]
Or maybe go in order of visibility from top to bottom. Anyway, feel free to add such a command here: `src/vs/workbench/browser/parts/notifications/notificationsCommands.ts`
